src/main/java/org/shanerx/tradeshop/commands/CommandCaller.java

```diff
@@ -180,19 +160,4 @@ public boolean checkPerm() {
 
 		return true;
 	}
-
-	/**
-	 * Checks if the sender has the required permission
-	 *
-     * @param exec Command attempting to be run
-     * @return true if permission is NONE or sender has permission
-	 */
-	public boolean checkPerm(TradeCommand exec) {
```

Comment:
I think this being removed could cause a problem, does it show the checkperm usage on line 64 as having a reference?

Reply:
This method was originally part of the custom command part of the tsaf. The class “TradeCommand” was removed in my last commit, since it was meant to represent custom subcommands for /tradeshop
Either way, my IDE did a quick search and all the instances of it being called I removed